### PR TITLE
feat(perception_online_evaluator): unify debug markers instead of separating for each object

### PR DESCRIPTION
**Pull request by kosuke55**
## Description

making debug marker for each uuid objects is heavy, so unify the markers.

## Tests performed

psim

image

image

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: evaluator/perception_online_evaluator/src/perception_online_evaluator_node.cpp [hunk 1]
```diff
@@ -132,50 +132,76 @@ void PerceptionOnlineEvaluatorNode::publishDebugMarker()
...
-      const auto c = createColorFromString(uuid + "_raw");
-      if (p.show_history_path) {
-        add(createPointsMarkerArray(history_path.first, "history_path_" + uuid, 0, c.r, c.g, c.b));
+  // visualize history path
+  {
+    const auto history_path_map = metrics_calculator_.getHistoryPathMap();
+    int32_t history_path_first_id = 0;
+    int32_t smoothed_history_path_first_id = 0;
+    size_t i = 0;
+    for (auto it = history_path_map.begin(); it != history_path_map.end(); ++it) {
```
Comment (rej55) [inline evaluator/perception_online_evaluator/src/perception_online_evaluator_node.cpp:143]: Can you change the format like following:
```
for (const auto & [uuid, histry_path] : history_path_map) {
...
}
```

File: evaluator/perception_online_evaluator/src/perception_online_evaluator_node.cpp [hunk 2]
```diff
@@ -132,50 +132,76 @@ void PerceptionOnlineEvaluatorNode::publishDebugMarker()
...
+  }
+
+  // visualize predicted path of past objects
+  {
+    int32_t predicted_path_first_id = 0;
+    int32_t history_path_first_id = 0;
+    int32_t deviation_lines_first_id = 0;
+    size_t i = 0;
+    const auto object_data_map = metrics_calculator_.getDebugObjectData();
+    for (auto it = object_data_map.begin(); it != object_data_map.end(); ++it) {
```
Comment (rej55) [inline evaluator/perception_online_evaluator/src/perception_online_evaluator_node.cpp:181]: Can you change the format like following:

```
for (const auto & [uuid, histry_path] : history_path_map) {
...
}
```

File: evaluator/perception_online_evaluator/src/perception_online_evaluator_node.cpp (hunk 1 above)
Comment (kosuke55) [reply evaluator/perception_online_evaluator/src/perception_online_evaluator_node.cpp]: fixed in 6ea8a0faa80fdece84e892d988c97b04d4998d83

File: evaluator/perception_online_evaluator/src/perception_online_evaluator_node.cpp (hunk 2 above)
Comment (kosuke55) [reply evaluator/perception_online_evaluator/src/perception_online_evaluator_node.cpp]: thanks, fixed in 6ea8a0faa80fdece84e892d988c97b04d4998d83

**Review by rej55 — APPROVED** [review]
LGTM

**Pull request merged**